desk/src/components/desk/global/AddNewTeamDialog.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+	<div>
+		<Dialog :options="{ title: 'Add New Team', size: 'sm' }" v-model="open">
+			<template #body-content>
+				<div class="space-y-4">
+					<div class="space-y-1">
+						<Input
+							label="Team Name"
+							type="text"
+							placeholder="Product Expert"
+							v-model="teamName"
+						/>
+					</div>
+					<div>
+						<label class="block text-sm leading-4 text-gray-700"
+							>Users</label
+						>
+						<VueMultiselect
```

Comment:
@fadilsid if you want you could take reference from the `Add Agent Dialog` which has a similar input

<img width="651" alt="image" src="https://user-images.githubusercontent.com/22856401/209657198-f50199c7-b8be-416a-8901-853a70c5ba89.png">

The component is currently hardcoded for adding new agents only, but you could take this as a reference